cli/README.md

```diff
@@ -2,13 +2,21 @@
 
 ## Usage
 
-### CLI
+### CLI Commands
+
+The Protego CLI supports the following commands:
+
+**1. List Spells (Default Command)**
```

Comment:
Let's not have a default command. It's more confusing.
Put the old default command under `@dewiz/xyz list`.

We don't need to keep backwards compatibility, so it is preferable that we are explicit.